$OpenBSD: patch-src_iconutils_cpp,v 1.3 2017/08/04 21:55:19 dcoppa Exp $

Use the Adwaita icon theme unconditionally: fixes a problem with
minitube GUI not having icons when executed outside of a Desktop
Environment

Do not use symbolic icons

Index: src/iconutils.cpp
--- src/iconutils.cpp.orig
+++ src/iconutils.cpp
@@ -23,11 +23,8 @@ $END_LICENSE */
 #include "mainwindow.h"
 
 QIcon IconUtils::fromTheme(const QString &name) {
-    const QLatin1String symbolic("-symbolic");
-    if (name.endsWith(symbolic)) return QIcon::fromTheme(name);
-    QIcon icon = QIcon::fromTheme(name + symbolic);
-    if (icon.isNull()) return QIcon::fromTheme(name);
-    return icon;
+    QIcon::setThemeName("Adwaita");
+    return QIcon::fromTheme(name);
 }
 
 QIcon IconUtils::fromResources(const QString &name) {
